consensus_pbft.go

```diff
@@ -290,6 +291,8 @@ func (p *Pbft) runAcceptState(ctx context.Context) { // start new round
 				p.setState(RoundChangeState)
 				return
 			}
+			// TODO: Maybe can be removed from here
+			p.msgQueue.processPendingCommitMessages(p.state.view.Copy(), p.verifyCommitMessage)
```

Comment:
Could we move this to the beginning of `ValidateState`? alongside the one in L366.

Reply:
Think we can, although wanted to validate commit message as early as possible (which is as soon as we set the proposal).

Reply:
Fixed https://github.com/0xPolygon/pbft-consensus/pull/78/commits/73c4d081b0a48e2c5a74773f783e9042f5d4ac2e